### DISCUSSION TITLE
Button not returning the full url

### DISCUSSION THREAD

**Discussion by Pixney-William** [transformstudios/statamic-review#11] [General]
So just bought this and i "think" i got it setup correctly. The fieldtype is added to the blueprint, i can use it and get the final part of the url. So its missing the "full url". But it is being set to : "https://....domaain.com". And the site is https. So should work?

But for some reason if i go into the list of collections, and click on review from there. It works. 

Any idea what i may do wrong here?

**Comment by edalzell** [comment]
Not quite sure what you mean? What is the exact URL it is being set to?

Also, can we see what your `sites.php`?

**Reply by Pixney-William** [reply]
The exact url, i don't want to write out. But it's with https and correct one (spec in APP_URL within the .env file)

**Comment by Pixney-William** [comment]
sites.php 

```
    'sites' => [

        'default' => [
            'name' => config('app.name'),
            'locale' => 'en_US',
            'url' => env('APP_URL', '/'),
        ],

    ],
```

**Comment by Pixney-William** [comment]
So to be more clear:

If i go to the listing of the collection. I can click on the three dot menu, and click on `review`. That opens the page with the APP_URL + review part.

If i go into the collection entry, and click on the review button i have added to the sidebar, i am only getting the review part of the url (/!...).

**Reply by Pixney-William** [reply]
I am on the 3.2.35 version of Statamic if that makes any difference.

**Comment by Pixney-William** [comment]
Sorry @edalzell ! Now i got it working. Must have been the cached configs even though i thought i cleared em.
